### DISCUSSION TITLE
next/link warning on strings inside <Link> should be more clear.

### DISCUSSION THREAD

**Discussion by daveteu** [vercel/next.js#16293] [Help]
Next/Link error message should be more clear. I have hundreds of components with <Link> used. Cannot identify which is using just string in <Link>

<img width="1979" alt="image" src="https://user-images.githubusercontent.com/18649392/90488127-008b4200-e16e-11ea-8e65-f2c10a7559d6.png">

## Describe the solution you'd like

Error should should where error occur, or offer a url with document to show how to find the <Link> components which String as child.

**Comment by maerzhase** [comment]
Thats intersting. Which version of next.js are you using? I am using `9.5.1` and i see the following stack trace. pointing to `Footer.js` where the wrong link is placed.

<img width="773" alt="Screenshot 2020-08-18 at 12 17 26" src="https://user-images.githubusercontent.com/1128485/90506913-e9882800-e14c-11ea-93b3-aacba0960a3c.png">

**Reply by daveteu** [reply]
You are using Chrome?

Turn out it will show up in Chrome, but not in Safari. Could be a bug.

**Reply by maerzhase** [reply]
Yes this screenshot is from chrome indeed! Interesting that safari doesn't resolve the stack. I can double check this again later.

**Reply by maerzhase** [reply]
You are right I can reproduce when the warning is thrown in a component, you can see the component name in chrome but not in safari.
